### ISSUE TITLE
[Branch] disable-srl16

### ISSUE THREAD

**Issue by mithro** [SymbiFlow/yosys#38]
### Why did we need this? (what does this change enable us to do)

Currently VPR doesn't have a model for the SRL16 primitive.

### What did it change?

Removed the SRL16 primitive.

### Should it be merged upstream - if not, when can we delete it?

Can be deleted when VPR supports the SRL16 primitive.

**Comment by eddiehung** [comment]
Does VPR support SRL32CE?

**Comment by litghost** [comment]
Yes!  And we are adding SRL16 shortly: https://github.com/SymbiFlow/symbiflow-arch-defs/pull/961
